docs/UserGuide.md

```diff
@@ -1289,7 +1288,7 @@ This section describes commands that help you manage your sessions.
 
 #### 📝Creating a Session: `createsession`
 
-You can create a session for any upcoming or past sessions.
+You can create a session for any upcoming or past sessions if you need to schedule or document class sessions in the system.
```

Comment:
You can create a session for any upcoming or past sessions if you need to schedule or document `your` class sessions in the system.